src/datachain/lib/dc/datachain.py

```diff
@@ -341,12 +343,14 @@ def settings(
                       To disable prefetching, set it to 0.
             namespace: namespace name.
             project: project name.
+            batch_rows: number of rows per batch (default=2000)
+            batch_mem: memory limit in MB per batch (default=1000)
```

Comment:
check / cleanup other entries in terms of style, compare with other docs (spaces before / after :, periods, etc)

Reply:
Yes. Its in draft for the same reason. I wanted to see the CI run.

Reply:
Updated it to same conventions within this docs. The style varies from docs to docs in our codebase. Either we need to prepare a guideline for docstrings and change the same for all. For now, it is out of scope of this PR.